src/sentry/runner/commands/devserver.py

```diff
@@ -403,6 +407,7 @@ def devserver(
                             "--consumer-group=sentry-consumer",
                             "--auto-offset-reset=latest",
                             "--no-strict-offset-reset",
+                            *((f"--dlq-topic={dlq_topics[name]}",) if name in dlq_topics else ()),
```

Comment:
Let's never DLQ in devserver. Better to hard error if something is bad as users may not realise something is broken when testing their code.

Reply:
I still think it's important to be able to test the dlq locally.

I moved the dlq topic to consumer definition, and changed the devserver command line option to include an `--enable-dlq [consumer name]` option.

So, when people use

```
sentry devserver --workers
```

The devserver behaves the same as before, and if they want to enable dlq for some consumers (say ingest-generic-metrics and ingest-metrics), they can use

```
sentry devserver --workers --enable-dlq ingest-generic-metrics --enable-dlq ingest-metrics
```